app/popup.js

```diff
@@ -477,6 +484,23 @@ class OptionsTab extends React.Component {
           </tbody>
         </table>
 
+        <h4 className="page-header">Import / Export</h4>
+        <div className="row">
+          <div className="col-xs-8">
+            <Button label='Export' clickHandler={_exportData} className='glyphicon-export'/>
+            <Button label='Import' clickHandler={() => {this.fileselector.click()}} className='glyphicon-import'/>
+            <input id="fileselector" type="file" onChange={_importData} ref={(input) => {this.fileselector = input}}/>
```

Comment:
Use `style={{ display: 'none }}` or a class name to hide this. We should stay away from `id` except for associating `label` elements with form inputs.